README.md

```diff
@@ -36,7 +36,10 @@ The following attributes are available for `viam:vision:motion-detector` vision
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
 | `cam_name` | string | **Required** | The name of the camera configured on your robot. |
-| `min_box_size` | int | **Required** | The size (in square pixels) of the smallest bounding box to allow. Relevant for GetDetections/GetDetectionsFromCamera only.
```

Comment:
Was this actually required? I think if you don't specify it, the minimum size is 0, and everything works anyway. Did I mistakenly introduce a breaking change?